modules/applications/pages/ui-builder/download-and-launch.adoc

```diff
@@ -30,22 +30,21 @@ include::example$ui-builder/development/docker-compose.yml[]
 
 INFO: You can override and configure all xref:ui-builder-docker-installation.adoc#environment-variables[environment variables] in the `docker-compose.yml` file.
 
-- Create a `nginx` configuration file `nginx.conf` with the content below:
+== Configure the bonita-ui-proxy image
 
-.Click to see the nginx configuration file
-[%collapsible]
-====
-[source, nginxconfiguration]
-----
-include::example$ui-builder/development/nginx.conf[]
-----
-====
+The *bonita-ui-proxy* image is based on the official Nginx Docker image and  included the nginx.conf.
```

Comment:
```suggestion
The *bonita-ui-proxy* image is based on the official Nginx Docker image and includes the nginx.conf.
```

Reply:
should we link the nginx.conf file used so that it help understanding how it works ?

Reply:
I'm afraid that we will not update the doc when we will modify the conf. What if we put a command to print the content of the nginx.conf inside the container?

Something like:
```
docker run --rm -it bonitasoft.jfrog.io/docker/bonita-ui-proxy cat /etc/nginx/templates/nginx.conf.template
```